requirements.txt

```diff
@@ -1,4 +1,5 @@
 jupyterhub>=1.2
 oauthenticator>=14.0.0
 jupyterhub-kubespawner>=4.3.0
+kubernetes
```

Comment:
should this be kubernetes-asyncio instead?

Reply:
Oh, I see. I guess it was just an issue with forgotten jupyterhub version "1.1.3" in k8s and this dependency is not needed at all.